src/main/java/blackjack/domain/BetMoney.java

```diff
@@ -0,0 +1,43 @@
+package blackjack.domain;
+
+public class BetMoney {
+
+    private static final String INVALID_RANGE_MESSAGE = "배팅 금액이 정상 범위가 아닙니다";
+    private static final int LOWER_BOUND = 0;
+
+    private final Integer value;
+
+    public BetMoney(final Integer value) {
+        checkBetMoneyRange(value);
+        this.value = value;
+    }
+
+    private void checkBetMoneyRange(final Integer betMoney) {
+        if (betMoney < LOWER_BOUND) {
+            throw new IllegalArgumentException(INVALID_RANGE_MESSAGE);
+        }
+    }
+
+    public Integer getValue() {
```

Comment:
```suggestion
    public int getValue() {
```

래핑 클래스를 사용할 필요가 없어보이네요.

Reply:
해당 메서드는 제거되었습니다 감사합니다 구구~!